.github/workflows/ci-update-workflow.yml

```diff
@@ -0,0 +1,63 @@
+name: ci-update-workflow
+
+on:
+  schedule:
+  # Run every hour.
+  - cron:  '0 * * * *'
+
+jobs:
+  update-go-versions:
+    runs-on: ubuntu-latest
+    steps:
+    - id: 'auth'
+      uses: 'google-github-actions/auth@v1'
+      with:
+        credentials_json: '${{ secrets.GCP_CREDENTIALS }}'
+
+    - name: 'Set up Cloud SDK'
+      uses: 'google-github-actions/setup-gcloud@v1'
+      with:
+        version: '423.0.0'
```

Comment:
Is there a way to use latest? This will become stale very soon.

Reply:
Yes, leaving it empty will be using latest by default. Updated.